libr/core/core.c

```diff
@@ -3154,14 +3154,17 @@ R_API bool r_core_serve(RCore *core, RIODesc *file) {
 				}
 				goto out_of_function;
 			}
-			switch ((ut8)cmd) {
+			// prevent overflow in uncontrolled allocation size
+			if (cmd > 255) {
+				cmd = 255;
```

Comment:
cmd is ut8, the only value possible here is 256. and i dont see why it should be handled specially if there's no valid packet for it

Reply:
Okay so we don't touch this lgtm error.

Reply:
ut8 max value is 255=0xff not 256=0x100 so it can't even be 256.